components/org.wso2.bfsi.consent.management.common/src/test/java/org/wso2/bfsi/consent/management/common/ConsentManagementConfigParserTests.java

```diff
@@ -88,6 +99,30 @@ public void testGetDataSourceName() {
     @Test(priority = 5)
     public void testGetConnectionVerificationTimeout() {
 
-            Assert.assertEquals(ConsentManagementConfigParser.getInstance().getConnectionVerificationTimeout(), 2);
+        Assert.assertEquals(ConsentManagementConfigParser.getInstance().getConnectionVerificationTimeout(), 2);
+    }
+
+    @Test(priority = 5)
+    public void testGetConsentValidationConfig() {
+
+        Assert.assertNotNull(ConsentManagementConfigParser.getInstance().getConsentValidationConfig());
+    }
+
+    @Test(priority = 5)
+    public void testGetConsentCacheAccessExpiry() {
+
+        Assert.assertEquals(ConsentManagementConfigParser.getInstance().getConsentCacheAccessExpiry(), 60);
+    }
+
+    @Test(priority = 5)
+    public void testGetConsentCacheModifiedExpiry() {
+
+        Assert.assertEquals(ConsentManagementConfigParser.getInstance().getConsentCacheModifiedExpiry(), 60);
+    }
+
+    @Test(priority = 5)
```

Comment:
why is priority 5 for all test methods?

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/5/commits/974586914061af019b6c95ec62c09d8e61396097